NuKeeper/Configuration/CommandLineArguments.cs

```diff
@@ -8,7 +9,8 @@ public class CommandLineArguments
         [CommandLine("mode", "m"), Required]
         public string Mode;
 
-        [CommandLine("github_token", "t"), Required, SensitiveInformation]
+        [Environment("NuKeeper_github_token"), Required, SensitiveInformation]
+        [OverriddenBy(ConfigurationSources.CommandLine, "t")]
```

Comment:
Agreed that it's only the command line where "short names" is even a common thing. I'm wondering if we want them there.